doc/source/examples/networktheory/IEEEP370 Deembedding.ipynb

```diff
@@ -33,12 +48,10 @@
    "source": [
     "import skrf as rf\n",
     "import matplotlib.pyplot as plt\n",
-    "from skrf.calibration import IEEEP370_SE_NZC_2xThru\n",
-    "from skrf.calibration import IEEEP370_MM_NZC_2xThru\n",
-    "from skrf.calibration import IEEEP370_SE_ZC_2xThru\n",
-    "from skrf.calibration import IEEEP370_MM_ZC_2xThru\n",
-    "from skrf.media import MLine\n",
     "import numpy as np\n",
+    "from skrf.calibration import (IEEEP370_SE_NZC_2xThru, IEEEP370_MM_NZC_2xThru,\n",
```

Comment:
I suggest changing `skrf.calibration` to `skrf.deembedding`. In scikit-rf, we clearly distinguish between calibration (almost complete error correction at the VNA measurement plane) and deembedding (partial error correction to remove fixture effects).

IEEEP370 is a deembedding method, not a calibration method. Thus, it should be imported from `skrf.deembedding`, not `skrf.calibration`. The fact that `skrf.calibration` also works is only an implementation detail and goes against the official API documentation, which says it's located under `skrf.deembedding`, thus importing from `skrf.calibration` should be discouraged in the *official* documentation.

 (I already opened a PR #696 but I found it creates a conflict with this PR, so it's better to change it here instead).